tests/integration-tests/tests/ultraserver/test_gb200/test_gb200/91_nvidia_imex_prolog.sh

```diff
@@ -13,12 +13,16 @@ WAIT_TIME_TO_STABILIZE=30
 ALLOWED_INSTANCE_TYPES="^(p6e-gb200|g5g)"
 IMEX_SERVICE="nvidia-imex"
 
+# Global service lock to prevent concurrent service operations
+SERVICE_LOCK_FILE="/var/lock/nvidia-imex-service.lock"
+SERVICE_LOCK_TIMEOUT=120
```

Comment:
2min seems too much for a lock to interact with imex. How did you come with this time?